eventuals/event-loop.cc

```diff
@@ -350,6 +352,8 @@ void EventLoop::Check() {
       ////////////////////////////////////////////////////
 
       CHECK_EQ(context, Context::Switch(std::move(previous)).get());
+      CHECK(context->in_use_.load() > 0);
+      context->in_use_.fetch_sub(1);
```

Comment:
```suggestion
      context->unuse();
```
You can put the `CHECK` in the `unuse()` method.